tomipaul/Onboard
# #152998495 Set up application server

#### What does this PR do?
Sets up application server
#### Description of Task to be completed?
- add package.json
- add babel configuration
- ignore appropriate folders/files
- add server entry point
#### How should this be manually tested?
- clone the repository
- run `npm install` to install dependencies
- run `npm run start:dev` to start application server
- On the browser, go to http://localhost:5000
- You should see a message that reads `Welcome to the onboard API`
#### What are the relevant pivotal tracker stories?
#152998495
